Add support for additional mensas

fried-ice opened the pull request.

fried-ice committed:
Add suport for additional mensas

MaxFleur commented on src/main.py:
```diff
@@ -408,9 +427,24 @@ def main():
     updater.dispatcher.add_handler(start_handler)
 
-    mensa_handler = CommandHandler('mensa', mensa)
+    mensa_handler = CommandHandler('mensa', alte_mensa)
```
Now that the `alte_mensa` command is there, does it really make sense to still include the normal `mensa` command? Imo it's some sort of a duplicate, maybe even ambiguous.

fried-ice replied:
The bot is still named AlteMensaBot.
It's only fair that we completely prefer the one and only Alte Mensa here IMHO.

MaxFleur commented on src/main.py:
```diff
@@ -85,14 +85,33 @@ def mensa(update, context):
             context.bot.send_photo(chat_id=update.message.chat_id, photo=img_url, caption=markdown_highlight_char + elem["name"] + markdown_highlight_char, parse_mode="Markdown")
 
 
+def alte_mensa(update, context):
```
It's not really clear where these numbers (4, 35 etc) come from. Maybe some more explanation (comments etc) or even a new variable for each number is more meaningful?

fried-ice replied:
I added a comment right on top of the mensa function.

I'd prefer not to introduce new variables for this purpose. IMHO the process is pretty easily understandable as it is right now.
No need to add more complexity.

fried-ice committed:
Fix wrong Siedepunkt mensa bot command

fried-ice committed:
Add source of mensa IDs in comments

So users know, where they have to look when they want to add new mensas
or fix existing ones.
